test/infrastructure/inmemory/pkg/runtime/cache/sync.go

```diff
@@ -50,10 +50,10 @@ func (c *cache) startSyncer(ctx context.Context) error {
 		c.syncQueue.ShutDown()
 	}()
 
-	syncLoopStarted := false
+	syncLoopStarted := make(chan struct{})
```

Comment:
I think I would drop this entirely.

This now just checks that we got until l.56. I'm not sure I understand why we are waiting for that. At this point the only guarantee is that the log was written, which doesn't make sense

Reply:
Sounds good to me. I fixed it.